update_registry_database.py

```diff
@@ -0,0 +1,150 @@
+import os
+import yaml
+import pytest
+import pandas as pd
+from network_wrangler import ProjectCard
+
+
+def update_registry_database(
+    card_tuple: tuple, df: pd.DataFrame, config: dict, write_to_disk: bool
+) -> pd.DataFrame:
+    """
+    Returns an updated registry dataframe.
+    Args:
+        cards_tuple: a list of project cards and their filenames
+        df: input registry DataFrame
+        config: input configuration
+        write_cards: a boolean indicating whether project card updates should be written to disk
+    Returns:
+        Registry DataFrame updated
+
+    """
+
+    s_node = config_dict["start_node_number"]
+    s_link = config_dict["start_link_number"]
+
+    for card in card_tuple:
+        df, needs_updating, card_dict = update_nodes(df, card[0], s_node)
```

Comment:
be more specific about what this DF and dict  is. Is it `registry_updated_df` and `updated_card_dict`?